homework/index.js

```diff
@@ -29,19 +30,79 @@
     });
     return elem;
   }
+  function app(repos) {
+    const root = document.getElementById('root');
+    const container = createAndAppend('div', root, {
+      class: 'container',
+    });
+    const header = createAndAppend('header', container, {
+      class: 'header',
+    });
+    createAndAppend('p', header, {
+      text: 'HYF Repositories',
+    });
+    const select = createAndAppend('select', header, {
+      id: 'list',
+    });
+    const div = createAndAppend('div', root, { class: 'list-div' });
+    const list = createAndAppend('ul', div, { class: 'list1' });
+    const listContributor = createAndAppend('ul', div, { class: 'list2' });
+    const newList = createAndAppend('li', list, { text: ' ' });
+    const newList1 = createAndAppend('li', list, { text: ' ' });
+    const newList2 = createAndAppend('li', list, { text: ' ' });
+    const newList3 = createAndAppend('li', list, { text: ' ' });
 
+    for (let i = 0; i < repos.length; i++) {
+      createAndAppend('option', select, { text: repos[i].name, value: i });
+    }
+
+    function leftHand() {
+      newList.innerHTML = `Repository: <a target=_blank href= ${repos[select.value].html_url}>${
+        repos[select.value].name
+      }</a>`;
+      newList1.innerText = `Description: ${repos[select.value].description}`;
+      newList2.innerText = `Forks: ${repos[select.value].forks}`;
+      newList3.innerText = `Updated: ${repos[select.value].updated_at}`;
+      const links = repos[select.value].contributors_url;
+      fetchJSON(links, (err, data) => {
+        if (err) {
+          createAndAppend('div', root, { text: err.message, class: 'alert-error' });
+        } else {
+          let linksContributor = JSON.stringify(data, null, 2);
+          linksContributor = JSON.parse(linksContributor);
```

Comment:
Why are you stringify'ing and on the next line parsing it again, looks to me there is no need and you can just ust the `data` 💁‍♂️

Reply:
you are right no need to do that! I paied my attention to that in the homework week 2 so, I delet it. :)